brainio/lookup.py

```diff
@@ -54,36 +55,51 @@ def data():
 
 def list_stimulus_sets():
     stimuli_rows = data()[data()['lookup_type'] == TYPE_STIMULUS_SET]
-    return list(set(stimuli_rows['identifier']))
+    return sorted(list(set(stimuli_rows['identifier'])))
```

Comment:
this will return a generator (i.e. can only iterate once), I think swapping the order of `sorted` and `list` would be preferable

Reply:
Oops, thanks.

Reply:
Actually, not according to the Python docs and not according to my debugger.  

https://docs.python.org/3.7/library/functions.html#sorted

Reply:
oh sorry, I must have mis-remembered